### ISSUE TITLE
How to deal with Error on build

### ISSUE THREAD

**Issue by MinaRe** [bharath272/semantic_contours#1]
Hi @bharath272 

Thanks for sharing your great job,
I try to use MNC, I have problem to create .mat (inst, cls) file on my data set. I download (https://9bc0b5eb4c18f1fc9a28517a91305702c68a10ae.googledrive.com/host/0ByUkob0WA1-NQi1sNlg4WkJQbTg/codes/SBD/download.html)  based on your work. I have received an error on building. Can you please kindly help me.
![screenshot from 2016-07-27 16 19 09](https://cloud.githubusercontent.com/assets/17288488/17178535/254c1a90-5416-11e6-8906-974f78df12de.png)

**Comment by bharath272** [comment]
Do you want the boundary evaluation code? If not, then you do not need to build anything. Untaring should be enough.
Note boundary evaluation is not needed for MNC.

**Comment by MinaRe** [comment]
I think, for segmentation training by MNC as you can see in imdb.py, we need PascalVOCSDS dataset.

__sets = {
    'voc_2012_seg_train': (lambda: PascalVOCSeg('train', '2012', 'data/VOCdevkitSDS/')),
    'voc_2012_seg_val': (lambda: PascalVOCSeg('val', '2012', 'data/VOCdevkitSDS/')),
    'voc_2007_trainval': (lambda: PascalVOCDet('trainval', '2007')),
    'voc_2007_test': (lambda: PascalVOCDet('test', '2007'))
}

**Comment by bharath272** [comment]
Yes, but they are just mat files, you should not need any compilation for that. The code is only for computing performance on the boundary metric, which is definitely not what MNC does.

The mat files should just be there.

If you can't find them, can you tell me what it is you see if you ls after you untarred the dataset?
